querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -1265,6 +1265,15 @@ public void num_cast2() {
     }
 
     @Test
+    public void num_cast_dateTime() { //TODO: think of good name
```

Comment:
Naming seems consistent with the rest of the test class. I'd accept this change with the TODO removed.